livedata/src/main/java/io/getstream/chat/android/livedata/ChatNotificationConfigOffline.kt

```diff
@@ -1,45 +1,33 @@
 package io.getstream.chat.android.livedata
 
 import android.content.Context
-import androidx.work.Constraints
-import androidx.work.NetworkType
-import androidx.work.OneTimeWorkRequestBuilder
-import androidx.work.WorkManager
-import androidx.work.workDataOf
+import android.content.Intent
+import androidx.core.content.ContextCompat
 import com.google.firebase.messaging.RemoteMessage
-import io.getstream.chat.android.client.logger.ChatLogger
 import io.getstream.chat.android.client.notifications.options.ChatNotificationConfig
-import io.getstream.chat.android.livedata.worker.SyncWorker
+import io.getstream.chat.android.livedata.service.SyncService
 
 /**
- * Extend the ChatNotificationConfig to start the sync worker when receiving new messages
+ * Extend the ChatNotificationConfig to start the sync service when receiving new messages
  * This ensures that offline storage is up to date as soon as you open the app
  */
 open class ChatNotificationConfigOffline(context: Context) : ChatNotificationConfig(context) {
-    private val logger = ChatLogger.get("ChatDomain ChatNotificationConfigOffline")
 
     override fun onFirebaseMessage(message: RemoteMessage): Boolean {
-        logger.logI("onFirebaseMessage received a message")
-
-        val data = message.getData()
-
+        val data = message.data
         val channelId = data.get("channel_id").toString()
         val channelType = data.get("channel_type").toString()
         val cid = "$channelType:$channelId"
 
-        logger.logI("onFirebaseMessage started a worker to update channel $cid")
-        //  Start background work manager
-        val constraints = Constraints.Builder()
-                .setRequiredNetworkType(NetworkType.CONNECTED)
-                .build()
+        Intent(context, SyncService::class.java).apply {
```

Comment:
What do you think about to move "Intent creation" into `SyncService`?
Following this approach we can update the needed attributes to be instanciated and be sure all place where it is used add them.

You can create an static method into this class like:
```
fun createIntent(context: Context, cid: String): Intent = Intent(context, Service::class.java).apply { putExtra(EXTRA_CID, cid) }
```

It allows us to not need to share the "attribute name" on different parts of our code, and the service knows exactly which attributes expects